docs/developers/how-to/testing.rst

```diff
@@ -0,0 +1,103 @@
+=============
+Running tests
```

Comment:
"Run MicroOVN tests" ?

no gerunds :)

Reply:
Oh, I thought tit applies only to page names. ack, no gerunds :D

Reply:
right, the policy applies to both headers and page names. the latter are actually top-level headers anyway (like in this case)